components/hearts.tsx

```diff
@@ -34,11 +34,14 @@ interface Props {
 export default function Hearts(props: Props) {
   const { lives } = props;
 
+  const elements: JSX.Element[] = [];
+  for (let counter = 1; counter <= lives; counter++) {
```

Comment:
Hello! Thank you for your contribution. I also want to have a flexible number of lives, but it's a bit more complicated than you patch. What you're doing here is to add the same number of component Heart that the player has lives, but this isn't what we want: we want the hearts to still be displayed, but differently when we lost a life:
![image](https://github.com/tom-james-watson/wikitrivia/assets/930064/43b20353-1540-48c5-8ea8-070ddcdfc4fe)

The actual number of lives is initialized here: https://github.com/tom-james-watson/wikitrivia/blob/master/lib/create-state.ts#L15
But as you spotted, we also need to have the number of `<Heart>` below dynamic.
So, we have to have the number of lives coming from somewhere. Should it be a const in a config file? Should we ask the users? Or let them select a level? (hard 3 lives, medium 5 lives, easy 8 lives?). What do you want?